multiapps-controller-persistence/src/main/java/org/cloudfoundry/multiapps/controller/persistence/query/AccessTokenQuery.java

```diff
@@ -0,0 +1,19 @@
+package org.cloudfoundry.multiapps.controller.persistence.query;
+
+import java.time.LocalDateTime;
+
+import org.cloudfoundry.multiapps.controller.persistence.OrderDirection;
+import org.cloudfoundry.multiapps.controller.persistence.model.AccessToken;
+
+public interface AccessTokenQuery extends Query<AccessToken, AccessTokenQuery> {
+
+    AccessTokenQuery id(Long id);
+
+    AccessTokenQuery value(byte[] value);
+
+    AccessTokenQuery username(String username);
+
+    AccessTokenQuery olderThan(LocalDateTime expiresAt);
```

Comment:
This could be renamed to expiredBefore

Reply:
Renamed to expiresBefore because it contains a verb